client/src/components/UIPatterns/UIPatternsSection/UIPatternsSection.js

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Section,
+  SectionTable,
+  DescriptionList,
+  InfiniteScrollTable,
+  InlineMessage,
+  SectionMessage
+} from '_common';
+import { useDispatch } from 'react-redux';
+import { Link, Route, Switch } from 'react-router-dom';
+
+import * as ROUTES from '../../../constants/routes';
+
+import { TicketCreateModal } from '../../Tickets';
+
+import './UIPatternsSection.module.css';
+
+const modalPath = `${ROUTES.WORKBENCH}${ROUTES.UI}/modal`;
+
+function UIPatternsSection() {
+  const dispatch = useDispatch();
+
+  function performAction() {
+    // eslint-disable-next-line no-alert
+    window.alert(
+      'Sample action (no actual action has occurred beyond this alert).'
+    );
+  }
+
+  return (
+    <dl>
+      <dt>
+        Minimal
+        <DescriptionList
+          className="small"
```

Comment:
Do we have some global definition for "small" that we should be thinking of using everywhere or is this just a classname for ui-patterns to target these sections?

Reply:
We do. That [global class `.small` is provided by Bootstrap](https://getbootstrap.com/docs/4.0/content/typography/#inline-text-elements). It is not specific to ui-patterns. I had concerns about this, also:

1. Do I rely on propagate the reliance on Bootstrap?
2. Or do I ignore pre-existing global and consistent `.small` (and matching `<small>` styles) in favor of a style specific to ui-patterns?

What do you think?

Reply:
I went with the dryer option.

[I replaced multiple uses of global Bootstrap class with single style (and reference to source of value](https://github.com/TACC/Core-Portal/pull/347/commits/73baba32b5f029bb1d75371f4162fe17391bd42d).